common/static/common/js/discussion/views/discussion_content_view.js

```diff
@@ -434,9 +434,9 @@
                 is_pinning = !this.model.get('pinned');
                 url = this.model.urlFor(is_pinning ? 'pinThread' : 'unPinThread');
                 if (is_pinning) {
-                    msg = gettext('We had some trouble pinning this thread. Please try again.');
+                    msg = gettext('We had some trouble pinning this post. Please try again.');
                 } else {
-                    msg = gettext('We had some trouble unpinning this thread. Please try again.');
+                    msg = gettext('We had some trouble unpinning this post. Please try again.');
```

Comment:
@alisan617 @andy-armstrong Can we also apply similar rewording to the couple of other error messages in this file? (I realize it's adding to scope!) Let's replace "We had some trouble..." with "...could not be...", and also remove any instances of "Please".
For example, in lines below:
"Your flag on this post could not be removed. Refresh the page and try again."
"This post could not be reported. Refresh the page and try again."